### DISCUSSION TITLE
RimWorld Japanese Steam Store Translations

### DISCUSSION THREAD

**Discussion by TiaPixel** [Ludeon/RimWorld-ja#52] [Q&A]
Hello! I'm Tia (TiaPixel), I'm helping Ludeon Studios with translations for the Ideology and Royalty Steam store pages.

I was looking for Japanese translators who would be interested in working with me to translate the English store text to Japanese for Ideology and Royalty. Please respond with a way I can contact you as well - ideally Discord! ✨💛

- Tia

**Comment by NM-Grack** [comment]
Hello. Thank you for contacting me.
I can talk on Discord, so please register the following tags! 

-Grack#5841

**Reply by TiaPixel** [reply]
Thanks for the response, I sent a friend request!
